README.rst

```diff
@@ -1,5 +1,5 @@
-schedule
-========
+`schedule <https://schedule.readthedocs.io/>`__
```

Comment:
Capitalize to Schedule?

Reply:
I guess `schedule` is fine :slightly_smiling_face: